src/Framework/Constraint/IsIdentical.php

```diff
@@ -55,10 +55,15 @@ public function __construct($value)
      */
     public function evaluate($other, string $description = '', bool $returnResult = false)
     {
+        $epsilon = self::EPSILON;
+        if (defined('CUSTOM_EPSILON')) {
```

Comment:
What about defining `EPSILON` conditionally in a namespace, and always having it declared?

Basides that, while the global state is immutable, I'm not sure I like the approach. What alternatives (if any) do we have?